packages/block-library/src/latest-posts/index.php

```diff
@@ -21,6 +43,9 @@ function render_block_core_latest_posts( $attributes ) {
 		'suppress_filters' => false,
 	);
 
+	$block_core_latest_posts_excerpt = $attributes['excerptLength'];
+	add_filter( 'excerpt_length', 'get_block_core_latest_posts_excerpt', 999 );
```

Comment:
I remember it from the docs of the `excerpt_length` filter: 

> Make sure to set the priority correctly, such as 999, otherwise the default WordPress filter on this function will run last and override what you set here.

Reply:
That would be surprising to me. The default filter is applied in `default-filters.php`:

https://github.com/WordPress/wordpress-develop/blob/822167a42b1a81bd94cf4c1e4ecaf958dc5233ae/src/wp-includes/default-filters.php#L185

As long as the code which adds the filter executes after this file (which I think is most common?), if it uses the same priority, it should take precedent over the default.

In any case, I don't think it needs to be quite so strong. If the primary goal is to take effect after the default, any value `11` or higher can suffice. Using some sensible increment (e.g. by `10`) can allow plugins to still insert "between" as well. A value like `20` might be a reasonable alternative.

Reply:
I've set this to 20, although only because 999 did indeed sound like the new z-index wars. Otherwise, it should be the last one applied as we want the rendering of this block to respect whatever that attribute is set to.